src/org/traccar/model/Command.java

```diff
@@ -77,6 +77,11 @@
     public static final String KEY_SERVER = "server";
     public static final String KEY_PORT = "port";
 
+    @Override
+    public Command clone() throws CloneNotSupportedException {
```

Comment:
Do we really need this override? I suggest we just use default clone as it is.

Reply:
It is `protected` https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#clone()

Reply:
I see. OK, can you then please fix indentation in this method.